content/components/banner.mdx

```diff
@@ -1,15 +1,14 @@
 ---
 title: Banner
-description: Use Banner to highlight important information.
-railsIds:
+description: Banner is used to highlight important information.
+componentId: banner
+railsIds: 
 - Primer::Alpha::Banner
 ---
 
 import ComponentLayout from '~/src/layouts/component-layout'
-import {Text} from '@primer/react'
+import {Link, Text} from '@primer/react'
+import {Link as GatsbyLink} from 'gatsby'
 export default ComponentLayout
 
-<Note variant="warning">
-  <Text sx={{display: 'block', fontWeight: 'bold', mb: 2}}>Work in progress</Text>
-  We are currently working on general guidance for this component.
-</Note>
+For general design and usage guidelines of banners, see the [flash component](/components/flash).
```

Comment:
Should we file an issue to rename "Banner" to "Flash" in PVC?

cc @langermank because she's done some component API consistency work in the past

Reply:
pasting this message here from a DM:

If you click on the [Rails tab of the flash component](https://primer.style/design/components/flash/rails/beta), it appears there actually is a component in rails named flash, and it's identical to banner.

Was banner the old component and the new one is flash (hence, beta?)

Reply:
Ah yeah, sorry about the confusion with these two components. The flash mechanism is a Rails thing, so the original component was named `Flash`. Flashes appear at the top of the page after you take some sort of action, like submit a form. Unfortunately, engineers working in dotcom started using the `Flash` component for all sorts of inappropriate things, i.e. not displaying flash messages (which again are a very specific Rails thing).

The `Banner` component is designed for these non-`Flash` use-cases. Whereas `Flash`es should announce themselves to screen readers, `Banner`s should not. Whereas `Flash`es should be designed to appear at the top of the page, `Banner`s can appear in-context. In other words, they should behave differently and therefore be used differently.

I would also argue the term `Flash` doesn't make sense outside of Rails. We should probably use the term "banner" in PRC.